samples/python/bulk_edit_issues.py

```diff
@@ -0,0 +1,47 @@
+import requests
+import json
+
+page_size = 1000 #max value
+page = 0
+api_key = "<api-key>"
+endpoint = "https://api.helpshift.com/v1/<domain>/issues"
+
+def bulk_edit_issue(issues):
+    print ("Triggering Bulk Action for " + str(len(issues)) + " issues")
+    api_params = {"issue-ids": json.dumps([issue['id'] for issue in issues]),
+                  "status": "Resolved",
+                  "message-body": "Your order will reach you soon!"}
+    response = requests.put(endpoint,
+                            params=api_params,
+                            auth=(api_key, ""))
+    response.raise_for_status()
+
+
+while True:
+    page += 1
```

Comment:
- The initial value of the page should be `1`.
- Increment to the page should be done at the end of the loop.

Reply:
Could be done that way too. This was to keep it consistent with the other code available in this folder.

Reply:
Benefits of doing it that way:
- The initial value of `page` would be `1` which is a valid value.
- This will follow the loop methodology, ie, run->iterate->run

Reply:
Yeah, I agree with that. I've fixed it in my commit. 
I'll add conventions document for this so that the other one is not followed.